snap/hooks/install

```diff
@@ -45,7 +45,9 @@ function set_base_config_props () {
 create_file_structure
 set_base_config_props
 
-chown -R snap_daemon "${SNAP_DATA}/"*
-chgrp root "${SNAP_DATA}/"*
-chown -R snap_daemon "${SNAP_COMMON}/"*
-chgrp root "${SNAP_COMMON}/"*
+#  "${SNAP_COMMON}"
+declare -a folders=("${SNAP_DATA}")
```

Comment:
just Q: why not add snap_common?

Reply:
Because the only process writing into `snap_common` is the opensearch daemon, which starts as the `snap_daemon` user, so all the directories and files are owned by `snap_daemon` and not root - which would make this fail